### PR TITLE
Docs lint and unify

### PR DESCRIPTION
**Pull request by roysagi**
Updated the flags in lint command in readme.
Took the examples to the separate docs and expanded the lint examples (nothing to expand in unify...). 

@reutshal - FYI

### CODE REVIEW

File: docs/lint_command.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+### Lint
+
+Run lintings (flake8, mypy, pylint, bandit) and pytest.
+pylint and pytest will run within all the docker images of an integration/script. Meant to be used with integrations/scripts that use the folder (package) structure. Will lookup up what docker image to use and will setup the dev dependencies and file in the target folder.
```
Comment (ronykoz) [inline docs/lint_command.md]: Add a break line before this one. 
+
The appropriate docker image for the integration/script will be used to execute the pytest and pylint checks.

File: docs/unify_command.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+### Unify
```
Comment (ronykoz) [inline docs/unify_command.md]: Add an explanation for why and when they would like to use it.

File: docs/unify_command.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Unify the code, image and description files to a single Demisto yaml file.
+**Arguments**:
+* *-i INDIR, --indir INDIR*
+  The path to the directory in which the files reside
+* *-o OUTDIR, --outdir OUTDIR*
+  The path to the directory into which to write the unified yml file
+
+**Examples**:
+`demisto-sdk unify -i Integrations/MyInt -o Integrations`
```
Comment (ronykoz) [inline docs/unify_command.md]: add script example

File: docs/lint_command.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+### Lint
```
Comment (ronykoz) [inline docs/lint_command.md]: add an explanation for why do we want this
Make sure that the code is up with python standards, prevents bugs, runs testing, etc...

File: docs/lint_command.md (hunk 1 above)
Comment (roysagi) [reply docs/lint_command.md]: done

File: docs/unify_command.md (hunk 3 above)
Comment (roysagi) [reply docs/unify_command.md]: done

File: docs/unify_command.md (hunk 2 above)
Comment (roysagi) [reply docs/unify_command.md]: done

File: docs/lint_command.md (hunk 4 above)
Comment (roysagi) [reply docs/lint_command.md]: kinda nailed it with this explanation... I had nothing to add... 
Done

File: docs/unify_command.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+### Unify
+
+Unify the code, image and description files to a single Demisto yaml file.
+
+**Use Cases**:
+This command is used in order to create a unified yml file, able ot be uploaded to Demisto via the
```
Comment (dantavori) [inline docs/unify_command.md]:
```suggestion
This command is used in order to create a unified yml file, able to be uploaded to Demisto via the
```

**Pull request merged**